### PR TITLE
Drop e2e-harness from integration testing 

### PR DESCRIPTION
**Pull request by tomahawk28**
Dropping e2e-harness, e2etemplates dependency for opsctl changes.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,9 +3,9 @@ module github.com/giantswarm/certctl
 go 1.13
 
 require (
+	github.com/ghodss/yaml v1.0.0
+	github.com/giantswarm/apprclient v0.2.0
 	github.com/giantswarm/backoff v0.2.0
-	github.com/giantswarm/e2e-harness v0.2.0
-	github.com/giantswarm/e2etemplates v0.2.0
```
Comment (tomahawk28) [inline go.mod:8]: Dropping e2e-harness, e2eteamplate dependancy.

File: go.mod [hunk 2]
```diff
@@ -3,19 +3,24 @@ module github.com/giantswarm/certctl
...
 	github.com/giantswarm/micrologger v0.3.1
 	github.com/giantswarm/vaultrole v0.2.0
 	github.com/hashicorp/vault/api v1.0.4
 	github.com/juju/errgo v0.0.0-20140925100237-08cceb5d0b53
+	github.com/spf13/afero v1.2.2
 	github.com/spf13/cobra v0.0.5
-	k8s.io/apimachinery v0.16.6
+	k8s.io/api v0.17.2
+	k8s.io/apimachinery v0.17.2
+	k8s.io/helm v2.16.4+incompatible
```
Comment (rossf7) [inline go.mod:23]: Can we drop this too? 

Maybe we're using the helm options like `helm.ReleaseName` somewhere? These are now part of helmclient.

Comment (tomahawk28) [reply go.mod:23]: Good catch, let me search for that usage.

Comment (tomahawk28) [reply go.mod]: I found out I also need to remove `replace` section in go.mod. Thanks for notice!

**Pull request merged**